### ISSUE TITLE
Power and Current not updated for POW devices

### ISSUE THREAD

**Issue by Vards0** [AlexxIT/SonoffLAN#125]
Hi Alexx

I cannot seem to get the current power usage (W) from my POWR2 device after it is switched on.  The guage remains at 0.  

I presume this is either due to an incorrect configuration or has something to do with the refresh process.

My configuration.yaml settings for it are as follows:-

```
sonoff:
  username: !secret
  password: !secret
  mode: local
  reload: always
  scan_interval: 00:01:00  
  devices:
     switch.sonoff_10009xxxxx:
      name: POWR2
      force_update: True

sensor:
- platform: template
  sensors:
      power_pow:
      friendly_name: POWR2 Power Usage
      device_class: power
      value_template: "{{ state_attr('switch.sonoff_10009xxxxx', 'power') }}"
```

Any ideas?  I have the same config in place for a TH16 temperature sensor and that works perfectly.

Thanks

**Comment by AlexxIT** [comment]
Strange indentations. You shure what you copied it right? Because they have a lot of problems.

**Comment by Vards0** [comment]
Hi Alexx

I have re-copied it below.  My indentations were probably wrong on the first post.

sonoff:
  username: !secret
  password: !secret
  mode: local
  reload: always
  scan_interval: 00:01:00  # (optional) default 5 minutes
  devices:
    switch.sonoff_10009xxxxx:
      name: POWR2
      force_update: True

sensor:
- platform: template
  sensors:
    power_pow:
      friendly_name: POWR2
      device_class: power
      value_template: "{{ state_attr('switch.sonoff_10009xxxxx', 'power') }}"

**Comment by Axellum** [comment]
+1, the power and current is active when the socket is disconnected, but zero when the socket is connected.

My configuration.yaml settings:

sonoff:
  username: !secret axel_mail
  password: !secret sonoff_pass
  reload: always  # update device list every time HA starts
  default_class: light
  devices:
    1000cxxxxx:
      device_class: light
    1000bxxxx:
      device_class: light

sensor:
 - platform: template
   sensors:
     voltage:
       friendly_name: Voltage
       device_class: power
       value_template: "{{ state_attr('light.sonoff_1000bxxxx', 'voltage') }}"
     puissance:
       friendly_name: Puissance
       device_class: power
       value_template: "{{ state_attr('light.sonoff_1000bxxxx', 'power') }}"
     current:
       friendly_name: current
       device_class: power
       value_template: "{{ state_attr('light.sonoff_1000bxxxx', 'current') }}"
     supported_features:
       friendly_name: supported_features
       value_template: "{{ state_attr('light.sonoff_1000bxxxxx', 'supported_features') }}"

**Comment by AlexxIT** [comment]
It’s known bug of Pow firmware in local mode. Needs to send turn_on and turn_off command several times. I’ll think about add this logic in component.

**Comment by AlexxIT** [comment]
@Vards0 @Axellum please show logs with `scan_interval: 00:01:00` and `force_update: True`.
Are power data corrected in 1 minute after being turned on / off?
https://youtu.be/Lt5fT4N5Pm8

**Comment by Vards0** [comment]
Hi Alex.

Please see my logs below:-

`2020-05-29 09:43:09  DEBUG    main          SysInfo: {'version': '0.110.3', 'dev': False, 'hassio': False, 'virtualenv': False, 'python_version': '3.7.7', 'docker': True, 'arch': 'armv7l', 'os_name': 'Linux', 'os_version': '4.19.114-v7'}
2020-05-29 09:43:09  DEBUG    main          LOCAL mode start
2020-05-29 09:43:15  DEBUG    sonoff_cloud  15 devices loaded from the Cloud Server
2020-05-29 09:44:24  DEBUG    sonoff_local  1000915df9 => Local4 | {'switch': 'on'} <= {'seq': 18, 'sequence': '1590738264184', 'error': 0}
2020-05-29 09:44:24  DEBUG    sonoff_local  1000915df9 <= Local3 | {'alarmVValue': [-1, -1], 'alarmCValue': [-1, -1], 'alarmPValue': [-1, -1], 'switch': 'on', 'startup': 'off', 'pulse': 'off', 'pulseWidth': 500, 'sledOnline': 'on', 'power': 0, 'voltage': 201.2, 'current': 0}
2020-05-29 09:44:25  DEBUG    sonoff_local  1000915df9 => Local4 | {'switch': 'off'} <= {'seq': 19, 'sequence': '1590738265691', 'error': 0}
2020-05-29 09:44:25  DEBUG    sonoff_local  1000915df9 <= Local3 | {'alarmVValue': [-1, -1], 'alarmCValue': [-1, -1], 'alarmPValue': [-1, -1], 'switch': 'off', 'startup': 'off', 'pulse': 'off', 'pulseWidth': 500, 'sledOnline': 'on', 'power': 750.88, 'voltage': 197.41, 'current': 12.27}
2020-05-29 09:44:27  DEBUG    sonoff_local  1000915df9 => Local4 | {'switch': 'on'} <= {'seq': 20, 'sequence': '1590738267005', 'error': 0}
2020-05-29 09:44:27  DEBUG    sonoff_local  1000915df9 <= Local3 | {'alarmVValue': [-1, -1], 'alarmCValue': [-1, -1], 'alarmPValue': [-1, -1], 'switch': 'on', 'startup': 'off', 'pulse': 'off', 'pulseWidth': 500, 'sledOnline': 'on', 'power': 0, 'voltage': 194.27, 'current': 0}
2020-05-29 09:44:28  DEBUG    sonoff_local  1000915df9 => Local4 | {'switch': 'off'} <= {'seq': 21, 'sequence': '1590738268266', 'error': 0}
2020-05-29 09:44:28  DEBUG    sonoff_local  1000915df9 <= Local3 | {'alarmVValue': [-1, -1], 'alarmCValue': [-1, -1], 'alarmPValue': [-1, -1], 'switch': 'off', 'startup': 'off', 'pulse': 'off', 'pulseWidth': 500, 'sledOnline': 'on', 'power': 963.25, 'voltage': 194.27, 'current': 0.35}
2020-05-29 09:44:30  DEBUG    sonoff_local  1000915df9 => Local4 | {'switch': 'on'} <= {'seq': 22, 'sequence': '1590738270628', 'error': 0}
2020-05-29 09:44:30  DEBUG    sonoff_local  1000915df9 <= Local3 | {'alarmVValue': [-1, -1], 'alarmCValue': [-1, -1], 'alarmPValue': [-1, -1], 'switch': 'on', 'startup': 'off', 'pulse': 'off', 'pulseWidth': 500, 'sledOnline': 'on', 'power': 0, 'voltage': 194.27, 'current': 0}
2020-05-29 09:44:32  DEBUG    sonoff_local  1000915df9 => Local4 | {'switch': 'off'} <= {'seq': 23, 'sequence': '1590738272491', 'error': 0}
2020-05-29 09:44:32  DEBUG    sonoff_local  1000915df9 <= Local3 | {'alarmVValue': [-1, -1], 'alarmCValue': [-1, -1], 'alarmPValue': [-1, -1], 'switch': 'off', 'startup': 'off', 'pulse': 'off', 'pulseWidth': 500, 'sledOnline': 'on', 'power': 748.18, 'voltage': 180.01, 'current': 4.33}`

The readings seem to be coming through but the guage card doesn't change at all.

**Comment by AlexxIT** [comment]
@Vards0 please enable `force_update`, switch pow only once and wait while status updated.

**Comment by Vards0** [comment]
Hi Alex
`sonoff:
  username: !secret
  password: !secret
  debug: True
  mode: local
  reload: always
  scan_interval: 00:01:00  # (optional) default 5 minutes
  devices:
    switch.sonoff_1000916178:
      name: Pool Pump
      force_update: True`

`sensor:
- platform: template
  sensors:
      power_pow:
      friendly_name: Pool Pump Power
      device_class: power
      value_template: "{{ state_attr('switch.sonoff_1000916178', 'power') }}"`

I switched the POW on at 11:09 am and ran it for about 5 minutes.  This is the only entry that was entered into the the log for that switch:-

`2020-05-29 11:10:03  DEBUG    sonoff_local  1000916178 <= Local1 | {'alarmVValue': [-1, -1], 'alarmCValue': [-1, -1], 'alarmPValue': [-1, -1], 'switch': 'on', 'startup': 'stay', 'pulse': 'off', 'pulseWidth': 500, 'sledOnline': 'on', 'power': 364.76, 'voltage': 200.29, 'current': 2.61}`

The 364.76 that you can see in the guage reading below was showing after running the pump for 6 hours last night.  It does not return to zero after switching the pump off.

![image](https://user-images.githubusercontent.com/4409210/83244555-c1b5a380-a19f-11ea-8b37-2cb8543765a3.png)

![image](https://user-images.githubusercontent.com/4409210/83244567-c9754800-a19f-11ea-857e-4f33452ae7f3.png)

![image](https://user-images.githubusercontent.com/4409210/83244671-ec9ff780-a19f-11ea-8f56-f4b6f0310e27.png)

I hope this gives you enough information.

Thanks

**Comment by AlexxIT** [comment]
@Vards0 wrong config. Need to use deviceid, not entity_id.

```
sonoff:
  username: !secret
  password: !secret
  debug: True
  mode: local
  reload: always
  force_update: [power]  # !!! SonoffLAN v2.1+
  scan_interval: 00:01:00  # (optional) default 5 minutes
  sensors: [power]  # !!! SonoffLAN v2.1+
  devices:
    1000916178:
      name: Pool Pump
```

**Comment by Vards0** [comment]
Thank you Alex. 

This was exaclty the issue.  I have only been using HA for a week so please excuse the noobie mistake.

Thanks so much for your help with this.

**Comment by Vards0** [comment]
Hi Alex.

Apologies for this.  I assumed you no longer needed the logs.

Please see the logs below.  The POW device id is 1000915df9

`2020-05-31 16:50:41  DEBUG    switch        Refresh device state 1000915df9
2020-05-31 16:50:41  DEBUG    switch        Refresh device state 100091616a
2020-05-31 16:50:41  DEBUG    sonoff_local  100091616a => Local4 | {'sledonline': 'on'} <= {'seq': 30, 'sequence': '1590936641055', 'error': 0}
2020-05-31 16:50:41  DEBUG    sonoff_local  100091616a <= Local3 | {'alarmVValue': [-1, -1], 'alarmCValue': [-1, -1], 'alarmPValue': [-1, -1], 'switch': 'off', 'startup': 'off', 'pulse': 'off', 'pulseWidth': 500, 'sledOnline': 'on', 'power': 0, 'voltage': 206.84, 'current': 0}
2020-05-31 16:50:41  DEBUG    sonoff_local  1000915df9 => Local4 | {'sledonline': 'on'} <= {'seq': 33, 'sequence': '1590936641042', 'error': 0}
2020-05-31 16:50:41  DEBUG    sonoff_local  1000915df9 <= Local3 | {'alarmVValue': [-1, -1], 'alarmCValue': [-1, -1], 'alarmPValue': [-1, -1], 'switch': 'off', 'startup': 'off', 'pulse': 'off', 'pulseWidth': 500, 'sledOnline': 'on', 'power': 0, 'voltage': 207.71, 'current': 0}
2020-05-31 16:51:01  DEBUG    sonoff_local  10007be89e <= Local3 | {'mainSwitch': 'off', 'deviceType': 'normal', 'targets': [{'targetHigh': 60, 'reaction': {'switch': 'off'}}, {'targetLow': 50, 'reaction': {'switch': 'on'}}], 'switch': 'off', 'startup': 'off', 'pulse': 'off', 'pulseWidth': 500, 'sledOnline': 'on', 'sensorType': 'DS18B20', 'humidity': 0, 'temperature': -2030.44}
2020-05-31 16:51:11  DEBUG    sonoff_local  10007be89e <= Local3 | {'mainSwitch': 'off', 'deviceType': 'normal', 'targets': [{'targetHigh': 60, 'reaction': {'switch': 'off'}}, {'targetLow': 50, 'reaction': {'switch': 'on'}}], 'switch': 'off', 'startup': 'off', 'pulse': 'off', 'pulseWidth': 500, 'sledOnline': 'on', 'sensorType': 'DS18B20', 'humidity': 0, 'temperature': 37.54}
2020-05-31 16:51:29  DEBUG    sonoff_local  1000666120 <= Local3 | {'sledOnline': 'off', 'arming': True, 'rssi': -53}
2020-05-31 16:51:31  DEBUG    sonoff_local  10007be89e <= Local3 | {'mainSwitch': 'off', 'deviceType': 'normal', 'targets': [{'targetHigh': 60, 'reaction': {'switch': 'off'}}, {'targetLow': 50, 'reaction': {'switch': 'on'}}], 'switch': 'off', 'startup': 'off', 'pulse': 'off', 'pulseWidth': 500, 'sledOnline': 'on', 'sensorType': 'DS18B20', 'humidity': 0, 'temperature': -2010.44}
2020-05-31 16:51:41  DEBUG    sonoff_local  10007be89e <= Local3 | {'mainSwitch': 'off', 'deviceType': 'normal', 'targets': [{'targetHigh': 60, 'reaction': {'switch': 'off'}}, {'targetLow': 50, 'reaction': {'switch': 'on'}}], 'switch': 'off', 'startup': 'off', 'pulse': 'off', 'pulseWidth': 500, 'sledOnline': 'on', 'sensorType': 'DS18B20', 'humidity': 0, 'temperature': 38.04}
2020-05-31 16:51:42  DEBUG    switch        Refresh device state 1000915df9
2020-05-31 16:51:42  DEBUG    switch        Refresh device state 100091616a
2020-05-31 16:51:42  DEBUG    sonoff_local  100091616a => Local4 | {'sledonline': 'on'} <= {'seq': 31, 'sequence': '1590936702085', 'error': 0}
2020-05-31 16:51:42  DEBUG    sonoff_local  1000915df9 => Local4 | {'sledonline': 'on'} <= {'seq': 34, 'sequence': '1590936702035', 'error': 0}
2020-05-31 16:51:42  DEBUG    sonoff_local  100091616a <= Local3 | {'alarmVValue': [-1, -1], 'alarmCValue': [-1, -1], 'alarmPValue': [-1, -1], 'switch': 'off', 'startup': 'off', 'pulse': 'off', 'pulseWidth': 500, 'sledOnline': 'on', 'power': 0, 'voltage': 205.98, 'current': 0}
2020-05-31 16:51:42  DEBUG    sonoff_local  1000915df9 <= Local3 | {'alarmVValue': [-1, -1], 'alarmCValue': [-1, -1], 'alarmPValue': [-1, -1], 'switch': 'off', 'startup': 'off', 'pulse': 'off', 'pulseWidth': 500, 'sledOnline': 'on', 'power': 0, 'voltage': 206.56, 'current': 0}
2020-05-31 16:52:06  DEBUG    sonoff_local  10007be89e <= Local3 | {'mainSwitch': 'off', 'deviceType': 'normal', 'targets': [{'targetHigh': 60, 'reaction': {'switch': 'off'}}, {'targetLow': 50, 'reaction': {'switch': 'on'}}], 'switch': 'off', 'startup': 'off', 'pulse': 'off', 'pulseWidth': 500, 'sledOnline': 'on', 'sensorType': 'DS18B20', 'humidity': 0, 'temperature': 19.04}
2020-05-31 16:52:31  DEBUG    sonoff_local  10007be89e <= Local3 | {'mainSwitch': 'off', 'deviceType': 'normal', 'targets': [{'targetHigh': 60, 'reaction': {'switch': 'off'}}, {'targetLow': 50, 'reaction': {'switch': 'on'}}], 'switch': 'off', 'startup': 'off', 'pulse': 'off', 'pulseWidth': 500, 'sledOnline': 'on', 'sensorType': 'errorType'}
2020-05-31 16:52:35  DEBUG    sonoff_local  1000666120 <= Local3 | {'sledOnline': 'off', 'arming': True, 'rssi': -54}
2020-05-31 16:52:41  DEBUG    sonoff_local  10007be89e <= Local3 | {'mainSwitch': 'off', 'deviceType': 'normal', 'targets': [{'targetHigh': 60, 'reaction': {'switch': 'off'}}, {'targetLow': 50, 'reaction': {'switch': 'on'}}], 'switch': 'off', 'startup': 'off', 'pulse': 'off', 'pulseWidth': 500, 'sledOnline': 'on', 'sensorType': 'DS18B20', 'humidity': 0, 'temperature': 38.04}
2020-05-31 16:52:42  DEBUG    switch        Refresh device state 1000915df9
2020-05-31 16:52:42  DEBUG    switch        Refresh device state 100091616a
2020-05-31 16:52:42  DEBUG    sonoff_local  100091616a => Local4 | {'sledonline': 'on'} <= {'seq': 32, 'sequence': '1590936762165', 'error': 0}
2020-05-31 16:52:42  DEBUG    sonoff_local  100091616a <= Local3 | {'alarmVValue': [-1, -1], 'alarmCValue': [-1, -1], 'alarmPValue': [-1, -1], 'switch': 'off', 'startup': 'off', 'pulse': 'off', 'pulseWidth': 500, 'sledOnline': 'on', 'power': 0, 'voltage': 207.29, 'current': 0}
2020-05-31 16:52:42  DEBUG    sonoff_local  1000915df9 => Local4 | {'sledonline': 'on'} <= {'seq': 35, 'sequence': '1590936762090', 'error': 0}
2020-05-31 16:52:42  DEBUG    sonoff_local  1000915df9 <= Local3 | {'alarmVValue': [-1, -1], 'alarmCValue': [-1, -1], 'alarmPValue': [-1, -1], 'switch': 'off', 'startup': 'off', 'pulse': 'off', 'pulseWidth': 500, 'sledOnline': 'on', 'power': 0, 'voltage': 208.41, 'current': 0}
2020-05-31 16:53:36  DEBUG    sonoff_local  10007be89e <= Local3 | {'mainSwitch': 'off', 'deviceType': 'normal', 'targets': [{'targetHigh': 60, 'reaction': {'switch': 'off'}}, {'targetLow': 50, 'reaction': {'switch': 'on'}}], 'switch': 'off', 'startup': 'off', 'pulse': 'off', 'pulseWidth': 500, 'sledOnline': 'on', 'sensorType': 'DS18B20', 'humidity': 0, 'temperature': -3052.94}
2020-05-31 16:53:41  DEBUG    sonoff_local  1000666120 <= Local3 | {'sledOnline': 'off', 'arming': True, 'rssi': -53}
2020-05-31 16:53:43  DEBUG    switch        Refresh device state 1000915df9
2020-05-31 16:53:43  DEBUG    switch        Refresh device state 100091616a
2020-05-31 16:53:43  DEBUG    sonoff_local  1000915df9 <= Local3 | {'alarmVValue': [-1, -1], 'alarmCValue': [-1, -1], 'alarmPValue': [-1, -1], 'switch': 'off', 'startup': 'off', 'pulse': 'off', 'pulseWidth': 500, 'sledOnline': 'on', 'power': 0, 'voltage': 207.71, 'current': 0}
2020-05-31 16:53:44  DEBUG    sonoff_local  100091616a <= Local3 | {'alarmVValue': [-1, -1], 'alarmCValue': [-1, -1], 'alarmPValue': [-1, -1], 'switch': 'off', 'startup': 'off', 'pulse': 'off', 'pulseWidth': 500, 'sledOnline': 'on', 'power': 0, 'voltage': 206.61, 'current': 0}
2020-05-31 16:53:44  DEBUG    sonoff_local  1000915df9 => Local4 | {'sledonline': 'on'} <= {'seq': 36, 'sequence': '1590936823077', 'error': 0}
2020-05-31 16:53:44  DEBUG    sonoff_local  100091616a => Local4 | {'sledonline': 'on'} <= {'seq': 33, 'sequence': '1590936823190', 'error': 0}
2020-05-31 16:53:51  DEBUG    sonoff_local  10007be89e <= Local3 | {'mainSwitch': 'off', 'deviceType': 'normal', 'targets': [{'targetHigh': 60, 'reaction': {'switch': 'off'}}, {'targetLow': 50, 'reaction': {'switch': 'on'}}], 'switch': 'off', 'startup': 'off', 'pulse': 'off', 'pulseWidth': 500, 'sledOnline': 'on', 'sensorType': 'DS18B20', 'humidity': 0, 'temperature': 38.04}
2020-05-31 16:54:01  DEBUG    sonoff_local  10007be89e <= Local3 | {'mainSwitch': 'off', 'deviceType': 'normal', 'targets': [{'targetHigh': 60, 'reaction': {'switch': 'off'}}, {'targetLow': 50, 'reaction': {'switch': 'on'}}], 'switch': 'off', 'startup': 'off', 'pulse': 'off', 'pulseWidth': 500, 'sledOnline': 'on', 'sensorType': 'DS18B20', 'humidity': 0, 'temperature': 0.01}
2020-05-31 16:54:11  DEBUG    sonoff_local  10007be89e <= Local3 | {'mainSwitch': 'off', 'deviceType': 'normal', 'targets': [{'targetHigh': 60, 'reaction': {'switch': 'off'}}, {'targetLow': 50, 'reaction': {'switch': 'on'}}], 'switch': 'off', 'startup': 'off', 'pulse': 'off', 'pulseWidth': 500, 'sledOnline': 'on', 'sensorType': 'DS18B20', 'humidity': 0, 'temperature': 38.54}
2020-05-31 16:54:13  DEBUG    sonoff_local  10009e9d1c <= Local3 | {'mainSwitch': 'off', 'deviceType': 'normal', 'targets': [{'targetHigh': 55, 'reaction': {'switch': 'off'}}, {'targetLow': 50, 'reaction': {'switch': 'on'}}], 'switch': 'off', 'startup': 'off', 'pulse': 'off', 'pulseWidth': 500, 'sledOnline': 'on', 'sensorType': 'DS18B20', 'humidity': 0, 'temperature': 61.98}
2020-05-31 16:54:15  DEBUG    sonoff_local  10009e9d1c => Local4 | {'switch': 'off', 'mainSwitch': 'off', 'deviceType': 'normal'} <= {'seq': 48, 'sequence': '1590936854528', 'error': 0}
2020-05-31 16:54:15  DEBUG    sonoff_local  10009e9d1c <= Local3 | {'mainSwitch': 'off', 'deviceType': 'normal', 'targets': [{'targetHigh': 55, 'reaction': {'switch': 'off'}}, {'targetLow': 50, 'reaction': {'switch': 'on'}}], 'switch': 'off', 'startup': 'off', 'pulse': 'off', 'pulseWidth': 500, 'sledOnline': 'on', 'sensorType': 'DS18B20', 'humidity': 0, 'temperature': 61.98}
2020-05-31 16:54:27  DEBUG    sonoff_local  1000915df9 => Local4 | {'switch': 'on'} <= {'seq': 37, 'sequence': '1590936867605', 'error': 0}
2020-05-31 16:54:27  DEBUG    sonoff_local  1000915df9 <= Local3 | {'alarmVValue': [-1, -1], 'alarmCValue': [-1, -1], 'alarmPValue': [-1, -1], 'switch': 'on', 'startup': 'off', 'pulse': 'off', 'pulseWidth': 500, 'sledOnline': 'on', 'power': 0, 'voltage': 209.88, 'current': 0}
2020-05-31 16:54:28  DEBUG    switch        Refresh device state 1000915df9
2020-05-31 16:54:28  DEBUG    sonoff_local  1000915df9 => Local4 | {'sledonline': 'on'} <= {'seq': 38, 'sequence': '1590936868115', 'error': 0}
2020-05-31 16:54:28  DEBUG    sonoff_local  1000915df9 <= Local3 | {'alarmVValue': [-1, -1], 'alarmCValue': [-1, -1], 'alarmPValue': [-1, -1], 'switch': 'on', 'startup': 'off', 'pulse': 'off', 'pulseWidth': 500, 'sledOnline': 'on', 'power': 0, 'voltage': 210.29, 'current': 0}
2020-05-31 16:54:34  DEBUG    sonoff_local  10009e9d1c <= Local3 | {'mainSwitch': 'off', 'deviceType': 'normal', 'targets': [{'targetHigh': 55, 'reaction': {'switch': 'off'}}, {'targetLow': 50, 'reaction': {'switch': 'on'}}], 'switch': 'off', 'startup': 'off', 'pulse': 'off', 'pulseWidth': 500, 'sledOnline': 'on', 'sensorType': 'DS18B20', 'humidity': 0, 'temperature': 61.23}
2020-05-31 16:54:44  DEBUG    switch        Refresh device state 1000915df9
2020-05-31 16:54:44  DEBUG    switch        Refresh device state 100091616a
2020-05-31 16:54:44  DEBUG    sonoff_local  100091616a => Local4 | {'sledonline': 'on'} <= {'seq': 34, 'sequence': '1590936884074', 'error': 0}
2020-05-31 16:54:44  DEBUG    sonoff_local  1000915df9 => Local4 | {'sledonline': 'on'} <= {'seq': 39, 'sequence': '1590936884031', 'error': 0}
2020-05-31 16:54:44  DEBUG    sonoff_local  100091616a <= Local3 | {'alarmVValue': [-1, -1], 'alarmCValue': [-1, -1], 'alarmPValue': [-1, -1], 'switch': 'off', 'startup': 'off', 'pulse': 'off', 'pulseWidth': 500, 'sledOnline': 'on', 'power': 0, 'voltage': 205.25, 'current': 0}
2020-05-31 16:54:44  DEBUG    sonoff_local  1000915df9 <= Local3 | {'alarmVValue': [-1, -1], 'alarmCValue': [-1, -1], 'alarmPValue': [-1, -1], 'switch': 'on', 'startup': 'off', 'pulse': 'off', 'pulseWidth': 500, 'sledOnline': 'on', 'power': 725.6, 'voltage': 204.52, 'current': 3.91}
2020-05-31 16:54:47  DEBUG    sonoff_local  1000666120 <= Local3 | {'sledOnline': 'off', 'arming': True, 'rssi': -53}
2020-05-31 16:54:53  DEBUG    sonoff_local  1000915df9 => Local4 | {'switch': 'off'} <= {'seq': 40, 'sequence': '1590936892264', 'error': 0}
2020-05-31 16:54:53  DEBUG    sonoff_local  1000915df9 <= Local3 | {'alarmVValue': [-1, -1], 'alarmCValue': [-1, -1], 'alarmPValue': [-1, -1], 'switch': 'off', 'startup': 'off', 'pulse': 'off', 'pulseWidth': 500, 'sledOnline': 'on', 'power': 726.27, 'voltage': 204.52, 'current': 3.91}
2020-05-31 16:54:53  DEBUG    switch        Refresh device state 1000915df9
2020-05-31 16:54:54  DEBUG    sonoff_local  1000915df9 => Local4 | {'sledonline': 'on'} <= {'seq': 41, 'sequence': '1590936893834', 'error': 0}
2020-05-31 16:54:54  DEBUG    sonoff_local  1000915df9 <= Local3 | {'alarmVValue': [-1, -1], 'alarmCValue': [-1, -1], 'alarmPValue': [-1, -1], 'switch': 'off', 'startup': 'off', 'pulse': 'off', 'pulseWidth': 500, 'sledOnline': 'on', 'power': 0, 'voltage': 205.15, 'current': 0}
2020-05-31 16:54:54  DEBUG    sonoff_local  10009e9d1c <= Local3 | {'mainSwitch': 'off', 'deviceType': 'normal', 'targets': [{'targetHigh': 55, 'reaction': {'switch': 'off'}}, {'targetLow': 50, 'reaction': {'switch': 'on'}}], 'switch': 'off', 'startup': 'off', 'pulse': 'off', 'pulseWidth': 500, 'sledOnline': 'on', 'sensorType': 'DS18B20', 'humidity': 0, 'temperature': 60.54}
`

Thanks

**Comment by AlexxIT** [comment]
`1000915df9` Sent all the time 0 as power... `force_update` did not help.

- Turn off `1000915df9`
- Go to `/developer-tools/service` page and try call `homeassistant.turn_on` service with `1000915df9` every 10 seconds several times
- And show the logs

**Comment by Vards0** [comment]
Hi Alex

I have called the homeassistant.turn_on service every 10 seconds and attached the resulting log file as requested.  The second time I called the service, the gauge card changed to reflect the power usage.

I've also attached a MP4 file that will hopefully also be of some use.  The first time the switch is turned on, the gauge card does not register the increase in watts.  However as soon as it is switched off, it will register the change and show it.  Turing it back on immediately it will also register the change and show it.

Thanks 

[PoolPump.zip](https://github.com/AlexxIT/SonoffLAN/files/4744517/PoolPump.zip)
[sonoff_debug_log.txt](https://github.com/AlexxIT/SonoffLAN/files/4744519/sonoff_debug_log.txt)

**Comment by AlexxIT** [comment]
I do not see this in the logs.
Not a problem, I ordered this device a couple of weeks ago. I will fix it when it arrives.

**Comment by Nix7git** [comment]
Anyone have resolved the problem?

**Comment by AlexxIT** [comment]
Probably fixed in latest master-version
